Lib/sqlite3/__main__.py

```diff
@@ -41,6 +41,13 @@ def execute(c, sql, suppress_errors=True, theme=theme_no_color):
             sys.exit(1)
 
 
+def _eof_key():
+    if sys.platform == "win32" and "idlelib.run" not in sys.modules:
+        return "CTRL-Z"
+    else:
+        return "CTRL-D"
```

Comment:
```suggestion
if sys.platform == "win32":
    eofkey = "CTRL-Z"
else:
    eofkey = "CTRL-D"
```
I don't know if there is a particular reason for the extended check (maybe Erlend knows?), but in general we just check for the platform (e.g. as you did in the test), so I suggest we simply here too. I also don't see need for adding a function, caching it will also have a minor performance benefit.

Reply:
That makes sense. Fixed.